STAC-extensions.yaml

```diff
@@ -629,15 +629,28 @@ components:
       schema:
         $ref: '#/components/schemas/collectionsArray'
       explode: false
-    query:
-      name: query
+    filter:
+      name: filter
       in: query
       description: >-
-        query for properties in items. Use the JSON form of the queryFilter used
-        in POST.
+        Filter for properties in items. Uses OGC's Common Query Language (CQL)
+        text (cql-text) from part 3 of OGC API - Features.
       required: false
       schema:
         type: string
+      style: form
+      explode: false
+    filter-lang:
+      name: filter-lang
+      in: query
+      required: false
+      schema:
+        type: string
+        enum:
+          - cql-text
+          - cql-json
+        default: cql-text
```

Comment:
For POST should it default to `cql-json`?

Reply:
I thought about it, and even had a note in the text. But wasn't sure how to even go about specifying it, so ended up just dropping it in favor of using the OGC API examples directly, and figured people could be explicit about it. If you have a good openapi way to add it that'd be great, it's beyond my current openapi skills. We could also just open an issue for it, in our repo and probably in OGC API CQL as well.

Reply:
There's surely a way to do that. Likely remove the default value here and then add it back again where the refs for this schema are with allOf.

Reply:
I've been spending way too long on this release, which I mostly just wanted to do to be able to move on to stac-spec 1.0 releases. This type of openapi tweaking takes me hours, since I don't really understand it all, so I'm just going to move this to an issue and do in beta.2 unless someone is able to volunteer to fix it.

Reply:
I've fixed the issue mentioned above and some other things that came to my attention, also tried to directly link to their CQL OpenAPI fragment. Should be fine for a first release, but will need some work in future betas.

Reply:
By the way, they are still changing the JSON structure quite a bit (see https://github.com/opengeospatial/ogcapi-features/issues/460), so a release of CQL with STAC could lead to quite some issues?!

Reply:
Yeah, I saw that :( I had thought it was a bit more solid. Also noticed that Clemens is still working on his full review: https://github.com/opengeospatial/ogcapi-features/pull/456

Unless they get 460 merged in the next day or so then I think we should release and leave out CQL.